src/pages/HomePage.js

```diff
@@ -1,65 +1,47 @@
-import React, { useState, useRef, useEffect } from "react";
-import { makeStyles } from "@material-ui/core";
+import React, { useState, useRef, useEffect, useCallback } from "react";
 import MainView from "../components/HomePage/MainView";
 import ApplicationMessagesNotifier from "../components/ApplicationMessages";
 import { smoothScrollToRef } from "../utils";
 import { useDesktop } from "../utils/media-queries";
 import ProductDescription from "../components/HomePage/ProductPlacementArea/ProductDescription";
 import SearchResultsWidget from "../components/HomePage/SearchResultsArea/SearchResultsWidget/SearchResultsWidget";
-
-const useStyles = ({ isMobile, showSearchResults }) => makeStyles(() => (
-    showSearchResults &&
-        {
-            search: {
-                scrollSnapAlign: "start",
-            },
-            productDescription: {
-                scrollSnapAlign: isMobile ? "center" : "end",
-            },
-            searchResults: {
-                scrollSnapAlign: "start",
-
-                // don't apply snap scroll to the bottom of the search
-                // results, so that it's possible to see the footer
-                scrollMarginBottom: "100vh",
-            },
-        }
-));
```

Comment:
Where did the styles go?

Reply:
These styles relate to the "scroll-snap" feature that was messing up this PR, I've removed them here because this is more prioritary but created an issue to see if we can bring that back, check #273